src/Shopsys/ShopBundle/DataFixtures/Demo/MultidomainArticleDataFixture.php

```diff
@@ -54,9 +54,7 @@ public function __construct(
      */
     public function load(ObjectManager $manager)
     {
-        foreach ($this->domain->getAllIdsExcludingFirstDomain() as $domainId) {
-            $this->loadForDomain($domainId);
-        }
+        $this->loadForDomain(2);
```

Comment:
I do not see any reason for this.

Reply:
else it failed on non existent csv files

Reply:
So you should create those files and prefix everything with (de) right?